src/devices/cpu/h8/h8_sci.h

```diff
@@ -63,14 +63,14 @@ class h8_sci_device : public device_t {
 		CLK_RX = 2
 	};
 
-	enum {
-		CLKM_INTERNAL_ASYNC,
-		CLKM_INTERNAL_ASYNC_OUT,
-		CLKM_EXTERNAL_ASYNC,
-		CLKM_EXTERNAL_RATE_ASYNC,
-		CLKM_INTERNAL_SYNC_OUT,
-		CLKM_EXTERNAL_SYNC,
-		CLKM_EXTERNAL_RATE_SYNC
+	enum class clock_mode {
+		INTERNAL_ASYNC,
+		INTERNAL_ASYNC_OUT,
+		EXTERNAL_ASYNC,
+		EXTERNAL_RATE_ASYNC,
+		INTERNAL_SYNC_OUT,
+		EXTERNAL_SYNC,
+		EXTERNAL_RATE_SYNC
 	};
```

Comment:
Please call this thing `clock_mode_t` or something so the name isn’t identical to the name of a data member (it isn’t in the global namespace so it's allowed to have a name ending with `_t`).

Reply:
Done.